appveyor.yml

```diff
@@ -3,21 +3,19 @@ version: '{build}'
 skip_tags: true
 
 environment:
+  JRUBY_OPTS: "--dev -J-Xmx1024M --debug"
   matrix:
-    - ruby_version: "21"
-    - ruby_version: "21-x64"
-    - ruby_version: "jruby-9.0.4.0"
+    - ruby_version: "Ruby21"
+    - ruby_version: "Ruby21-x64"
+    - ruby_version: "jruby-9.0.0.0"
```

Comment:
Only available jruby-9.0.0.0 on appveyor:
https://ci.appveyor.com/project/mtsmfm/active-model-serializers/build/1

> putting the JRUBY_OPTS globally?

Yeah, I went same way for travis but it is not smart :confused: 

https://github.com/rails-api/active_model_serializers/blob/master/.travis.yml#L25-L26

Should I apply patch for that on this PR or create another PR?